experimental/babel-preset-env/data/built-in-features.js

```diff
@@ -1,3 +1,22 @@
+const es5 = {
+  "es6.array.every": "Array methods / Array.prototype.every",
```

Comment:
Shouldn't these be `es5.array.*`? (I have never looked at `preset-env`'s source code, so I may be wrong)

Reply:
These are the names of the matching [core-js modules](https://github.com/zloirock/core-js/blob/master/es5/index.js)